datumaro/plugins/widerface_format.py

```diff
@@ -59,7 +59,8 @@ def _load_items(self, path):
                     if len(bbox_list) == 10:
                         i = 4
                         for attr in WiderFacePath.BBOX_ATTRIBUTES:
-                            attributes[attr] = int(bbox_list[i])
+                            if bbox_list[i] != '-':
+                                attributes[attr] = int(bbox_list[i])
```

Comment:
I suggest using `occluded` instead of `occlusion` for compatibility with other formats.